backend/app/tests/services/scheduled_inquiries/test_create_scheduled_inquiry.py

```diff
@@ -0,0 +1,52 @@
+from collections.abc import Generator
+
+import pytest
+from sqlmodel import Session
+
+from app.models import Inquiry, InquiryCreate, ScheduledInquiry
+from app.services import inquiries as inquiries_service
+from app.services import scheduled_inquiries as scheduled_inquiries_service
+
+
+@pytest.fixture
+def db_inquiry(db: Session) -> Generator[Inquiry]:
+    inquiry_in = InquiryCreate(text="I am test inquiry.")
+    db_inquiry = inquiries_service.create_inquiry(session=db, inquiry_in=inquiry_in)
+    yield db_inquiry
+    db.delete(db_inquiry)
+
+
+def test_create_when_no_scheduled_inquiries_exist_should_assign_rank_1(
+    db: Session, db_inquiry: Inquiry
+) -> None:
+    expected_rank = 1
+
+    scheduled_inquiry = scheduled_inquiries_service.create(
+        session=db, inquiry_id=db_inquiry.id
+    )
+
+    db_scheduled_inquiry = db.get(ScheduledInquiry, scheduled_inquiry.id)
+
+    if not db_scheduled_inquiry:
+        raise Exception()
+
+    assert db_scheduled_inquiry.rank == expected_rank
+
+
+def test_create_when_scheduled_inquiries_exist_should_assign_correct_rank(
```

Comment:
I'd be more specific. e.g. 

test_create_when_one_scheduled_inquiry_exists_should_assign_rank_two

Reply:
also, and not sure if this is a JS convention, but at least in C#/C++ we camelCased it, e.g.

create_whenOneScheduledInquiryExists_shouldAssignRankTwo

seems a bit more readable. but either way.

Reply:
Yeah it looks god awful. Maybe we should go with camel case.